code/modules/clothing/suits/f13factionarmor.dm

```diff
@@ -531,6 +531,10 @@
 	slowdown = 0.08
 	armor = list("melee" = 50, "bullet" = 60, "laser" = 50, "energy" = 25, "bomb" = 55, "bio" = 60, "rad" = 60, "fire" = 90, "acid" = 20, "wound" = 55)
 
+/obj/item/clothing/suit/armor/f13/rangercombat/Initialize()
```

Comment:
Woah, stealth NCR buffs?

Reply:
I told him to. Cent armor is platable, ranger vet gear isn't, for some reason.

Reply:
You wanted to do this, yet complain about it? What the dog doing.